string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,23 @@
+/* exported numVowels */
+// input: string
+// output: number of vowel characters in string
+// create reference data for all possible vowels
+// create storage to keep track of vowel count
+// look at each character letter
+// reference each letter and look at possible vowels
+// if vowel or capital vowel
+// add one to counter storage
+// return the number counter
+
+function numVowels(string){
+  var vowelsArr = ["a", "e", "i", "o", "u"];
+  var counter = 0;
+  for(var i = 0; i < string.length; i++) {
+    for(var x = 0; x < vowelsArr.length; x++){
+      if(string[i] === vowelsArr[x] || string[i] === vowelsArr[x].toUpperCase()){
```

Comment:
I would just add the capital letters to your array as well, so that you can simplify this conditional in the loop.

Reply:
Got it, I actually had the full array with capitals first but thought it'd be better to make an or statement for the capitals instead. The updated version does look a lot cleaner though, so I totally get it!